lib/msal-node/README.md

```diff
@@ -1,50 +1,154 @@
+
+The `@azure/msal-node` package has a dependency on `@azure/msal-common` package, which is the common engine for all future javascript based libraries.
+
+### OAuth grant types supported and upcoming:
+
+The current alpha version supports the below OAuth grant types:
+
+- [Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) with [PKCE](https://oauth.net/2/pkce/)
+- [Device Code Grant](https://oauth.net/2/grant-types/device-code/)
+- [Refresh Token Grant](https://oauth.net/2/grant-types/refresh-token/)
+- Silent Flow (user convenience flow to acquire a token silently)
+
+In the upcoming quarters we plan to add support for:
+
+- Client Credentials flow
+- Confidential client flow (authorization code flow with client secret)
+
+More details on different grant types supported by Microsoft authentication libraries in general can be found [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-authentication-flows)
+
+### App models supported:
+
+The app models supported with this library are:
+- Destop app that calls web APIs
+- Web app that calls web APIs (upcoming)
+- Web APIs that call web APIs (upcoming)
+- Daemon apps (upcoming)
+
+More details on APP models and the authentication flows that map to each of them can be found [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-flows-app-scenarios)
+
+## FAQ
+
+See [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/FAQ.md).
+
+## Releases
+
+*Expect us to detail our major and minor releases moving forward, while leaving out our patch releases.  Patch release notes can be found in our change log.*
+
+| Date | Release | Announcement | Main features |
+| ------| ------- | ---------| --------- |
+| July 13th, 2020 (Tentative) | @azure/msal-node v1.0.0-alpha.1 | No release notes yet | Full version of the `@azure/msal-node` package; relies on `@azure/msal-common` v1.0.0 |
+| July 6th, 2020  | @azure/msal-node v1.0.0-alpha.0| No release notes yet | Full version of the `@azure/msal-node` package; relies on `@azure/msal-common` v1.0.0-beta.4 |
+
+## Prerequisites
+
+Before using `@azure/msal-node` you will need to register your app in the azure portal to get a valid `clientId` for configuration, and to register the routes that your app will accept redirect traffic on if applicable. Currently we support the below app registrations for `@azure/msal-node`:
+
+- [Desktop app that calls web APIs: App registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-app-registration)
+- [Protected web API: App registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-protected-web-api-app-registration)
```

Comment:
I think this is for protecting APIs (which we don't support)

Reply:
Yes. I think it is confidential flow. Thanks.